### PR TITLE
Be less restrictive about redis gem version

### PR DESCRIPTION
**Pull request by rhruiz**

### CODE REVIEW

**Comment by rhruiz** [comment]
@fabioperrella @lorn @rafaellima

File: Gemfile.lock [hunk 1]
```diff
@@ -4,7 +4,7 @@ PATH
     heartcheck-redis (1.0.2)
       heartcheck (>= 1.0.0)
       net-telnet (~> 0.1.1)
-      redis (>= 3.2.0, < 3.4.0)
+      redis (>= 3.2.0, < 5)
```
Comment (fabioperrella) [inline Gemfile.lock:7]: Given this range of accepted redis versions, do you think it is a good idea to change https://github.com/locaweb/heartcheck-redis/blob/master/.travis.yml to run the specs with redis 3.x and 4.x ?

Comment (fabioperrella) [reply Gemfile.lock:7]: To do it, maybe it would be necessary to have 2 different Gemfile.lock

Comment (rhruiz) [reply Gemfile.lock:7]: We already have 2 gemfiles because of rack 2.0 not working with rubies < 2.2.something.

No problem adding another

**Comment by devjoaoGustavo** [comment]
By distraction, I just opened another [pull request](https://github.com/locaweb/heartcheck-redis/pull/5) about the same thing, but I think that something I made may be useful. Could You all guys take e a look?

Thanks

**Comment by rhruiz** [comment]
Let's work this in #5

**Pull request closed without merging**